### PR TITLE
[PAY-1613] Update titles + icons for Library

### PR DESCRIPTION
**Pull request by nicoback2**
### Description
Designs: https://www.figma.com/file/ClKssErI6tIMmZFwQifp9f/%F0%9F%92%B0-USDC-%2B-Payments%3A-Feature?type=design&node-id=1078-48999&mode=design&t=uou92wlmvtTKajUv-0

I moved some changes to follow-ons:
- Updating mobile web nav bar Library icon (this would involve moving from Lottie to Rive here, not too big of a lift but not trivial) https://linear.app/audius/issue/PAY-1648/follow-on-update-mobile-web-bottom-bar
- Renaming "Favorites" files to "Library" (wanted to avoid merge headaches since we're using a feature branch) https://linear.app/audius/issue/PAY-1649/follow-on-rename-favorites-files-to-library-web-mobile

Screenshot 2023-07-19 at 3 56 37 PM
Screenshot 2023-07-19 at 4 03 36 PM

mobilelib

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?
Tested on local web + mo web + ios sim
*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Review by dylanjeffers — APPROVED** [review]
wonderful, so design is good with us just changing from favorites to library as soon as next release? Oh wait i see you are merging into a feature branch, we just going to do that moving forward for this feature?

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

**Comment by nicoback2** [comment]
> wonderful, so design is good with us just changing from favorites to library as soon as next release? Oh wait i see you are merging into a feature branch, we just going to do that moving forward for this feature?

yee i decided to just do a feature branch per piazza's suggestion to avoid a big headache with splitting the library code up.

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--update-sidebar

File: packages/stems/src/assets/icons/iconLibrary.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M3.25 2.9375C3.25 3.45703 3.66797 3.875 4.1875 3.875H19.8125C20.332 3.875 20.75 3.45703 20.75 2.9375C20.75 2.41797 20.332 2 19.8125 2H4.1875C3.66797 2 3.25 2.41797 3.25 2.9375ZM3.875 8.25C3.33594 8.25 2.82422 8.48047 2.46875 8.88281C2.11329 9.28516 1.94922 9.82422 2.01563 10.3594L3.26563 20.3594C3.38282 21.2969 4.17969 22 5.125 22H18.875C19.8203 22 20.6172 21.2969 20.7344 20.3594L21.9844 10.3594C22.0508 9.82422 21.8867 9.28906 21.5313 8.88672C21.1758 8.48438 20.6641 8.25391 20.125 8.25391H3.875V8.25ZM19.1875 15.4375C19.1875 18.3711 15.9688 20.75 12 20.75C8.03125 20.75 4.8125 18.3711 4.8125 15.4375C4.8125 12.5039 8.03125 10.125 12 10.125C15.9688 10.125 19.1875 12.5039 19.1875 15.4375ZM12 16.6875C12.6914 16.6875 13.25 16.2695 13.25 15.75C13.25 15.2305 12.6914 14.8125 12 14.8125C11.3086 14.8125 10.75 15.2305 10.75 15.75C10.75 16.2695 11.3086 16.6875 12 16.6875ZM3.5625 5.125C3.04297 5.125 2.625 5.54297 2.625 6.0625C2.625 6.58203 3.04297 7 3.5625 7H20.4375C20.957 7 21.375 6.58203 21.375 6.0625C21.375 5.54297 20.957 5.125 20.4375 5.125H3.5625Z" fill="currentColor"/>
```
Comment (schottra) [inline packages/stems/src/assets/icons/iconLibrary.svg:2]: Noting the use of `currentColor` here. I just did that in another branch, but it breaks from the established pattern on `web` of setting `fill` on paths through CSS modules (requires using the `color` property instead).

Personally, I prefer this pattern and am all about it. But it also differs from mobile, where the icons use `#000` and `#f00` as placeholders which are replaced via React props using SVGR. I think there's a chance we _might_ standardize on that in the future.

cc @dylanjeffers

**Review by schottra — APPROVED** [review]
Looks good! And thank you for doing small PRs against a feature branch. This already feels easier than reviewing a huge change with a bunch of feature flagging.

**Pull request merged**